guides/public-app/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
Put the .gitignore at repo root to avoid duplicating it?

Reply:
I expect users will copy this folder and apply modifications. Example:

<img width="841" alt="image" src="https://github.com/CartoDB/viz-doc/assets/1161870/fa8bb76c-777c-44e5-9a75-acf36e99680b">